datumaro/components/annotation.py

```diff
@@ -38,42 +49,47 @@ def type(self) -> AnnotationType:
         return self._type # must be set in subclasses
 
     def wrap(self, **kwargs):
+        "Returns a modified copy of the object"
         return attr.evolve(self, **kwargs)
 
 @attrs(kw_only=True)
 class Categories:
-    attributes = attrib(factory=set, validator=default_if_none(set), eq=False)
+    """
+    A base class for annotation metainfo. It is supposed to include
+    dataset-wide metainfo like available labels, label colors,
+    label attributes etc.
+    """
+
+    attributes: Set[str] = attrib(
+        factory=set, validator=default_if_none(set), eq=False)
 
 @attrs
 class LabelCategories(Categories):
-    @attrs(repr_ns='LabelCategories')
```

Comment:
Please remove the one in `PointsCategories`, as well.

Reply:
Done